content/en/platform/corda/4.10/community/get-started/get-started-landing.md

```diff
@@ -40,15 +40,15 @@ If you are already using an open source version of Corda (Corda 4.1–Corda 4.9)
 * Download the `.zip` [file](https://download.corda.net/corda-community-edition/4.10/community-4.10.zip)
 * Use the Docker image available on [Docker Hub](https://hub.docker.com/repository/docker/corda/community).
 
-Follow the upgrade guides to make sure your [node](../../community/node-upgrade-notes.md) and [CorDapps](../../community/upgrading-cordapps.md) are upgraded correctly.
+Follow the upgrade guides to make sure your [node]({{< relref "../../community/node-upgrade-notes" >}}) and [CorDapps]({{< relref "../../community/upgrading-cordapps" >}}) are upgraded correctly.
 
 ## Next steps
 
-1. Familiarize yourself with the [Corda key concepts](../about-corda/corda-key-concepts.md). 
-2. Run a [sample CorDapp](../../community/tutorial-cordapp.md) to see Corda in action.
-4. [Build your own CorDapp](../../community/building-a-cordapp-index.md) from scratch. /
+1. Familiarize yourself with the [Corda key concepts]({{< relref "../about-corda/corda-key-concepts" >}}). 
+2. Run a [sample CorDapp]({{< relref "../../community/tutorial-cordapp" >}}) to see Corda in action.
+4. [Build your own CorDapp]({{< relref "../../community/building-a-cordapp-index" >}}) from scratch. /
```

Comment:
```suggestion
4. [Build your own CorDapp]({{< relref "../../community/building-a-cordapp-index" >}}) from scratch. 
```

Reply:
Thanks for flagging Nadine. No this particular one is still there on purpose. It is one of the other lesser-used link types identified, but this regex doesn't find it. I will write a new one for that type.